.github/workflows/build.yml

```diff
@@ -24,13 +24,17 @@ jobs:
       run: |
         chmod +x build.sh
         $GITHUB_WORKSPACE//build.sh -v
+    - name: Test with Nunit on Mono
+      run: $GITHUB_WORKSPACE//test.sh
```

Comment:
Do we need the full argument as per line 37 below?

Reply:
The integration tests need more work on non-windows platforms. 
I am planning to add it here: https://github.com/chocolatey/choco/pull/2435